### DISCUSSION TITLE
Q: 在next.js项目中使用waline，在开发环境下评论功能可以正常使用，但在生产环境下就会报错__VUE_PROD_DEVTOOLS__ is not defined

### DISCUSSION THREAD

**Discussion by lzb446703678** [walinejs/waline#2247] [Q&A] [answered]
### Describe the bug | 问题描述

在next.js项目中使用waline，在开发环境下评论功能可以正常使用，但在生产环境下就会报错__VUE_PROD_DEVTOOLS__ is not defined

### Website URL | 问题网站

101jc.com

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
@Mister-Hope 我印象中好像你在群里回复过这个问题，是依赖版本的问题吗？

---
@Mister-Hope I have the impression that you have replied to this question in the group. Is it a version-dependent issue?

**Comment by Mister-Hope** [comment, marked as answer]
把你的vue相关的东西全都升级到能升级的最高版本

---
Upgrade all your vue-related things to the highest version that can be upgraded
